protocol/contracts/amo/helpers/AMOCommon.sol

```diff
@@ -0,0 +1,21 @@
+pragma solidity ^0.8.4;
+// SPDX-License-Identifier: AGPL-3.0-or-later
+
+
+library AMOCommon {
+    error InvalidAddress();
+    error NotOperator();
+    error NotOperatorOrOwner();
+    error ZeroSwapLimit();
+    error AboveCappedAmount(uint256);
+    error InsufficientBPTAmount(uint256);
+    error InvalidBPSValue(uint256);
+    error RebalanceAmountTolerance(uint256, uint256);
+    error InsufficientAmountOutPostcall(uint256, uint256); //expected,actual
```

Comment:
You can (and probably should) add variable names to each of these custom errors to make it clearer
`error InsufficientAmountOutPostcall(uint256 expected, uint256 actual);`